waiverdb/templates/permissions.html

```diff
@@ -21,19 +21,23 @@
                     {% for p in permissions -%}
                     <tr>
                         <td>{{ p["name"] }}</td>
-                        <td>{{ p["maintainers"] | map("urlize") | join(" ") | safe }}</td>
-                        <td>{{ p["testcases"] | join("<br />") | replace('-', '&#8209;') | safe }}</td>
+                        <td><ul>
+                          {%- for maintainer in p["maintainers"] %}<li>{{ maintainer | urlize }}</li>{% endfor -%}
+                        </ul></td>
+                        <td><ul>
+                          {%- if p["_testcase_regex_pattern"] -%}<li><code>{{ p["_testcase_regex_pattern"] }}</code></li>{%- endif -%}
```

Comment:
would put something to show that this is a regex-pattern and not an fnmatch one (so symbols like `*` has different meaning). for example put regex between slashes: `/some regex/`

Reply:
Good idea.